tests/test_splitter.py

```diff
@@ -8,8 +8,57 @@
 
 class TestSplitter(TestCase):
 
-    def test_Split(self):
-        sp.Splitter.Split("filepath")
+    def test_Split_one(self):
+        # Arrange
+        filepath = os.getcwd() + "/../datasets/"
+        splitter = sp.Splitter();
+        splitter.Split((filepath + "testfile.csv"), ",")
 
-        self.assertEquals = (train, train)
-        self.assertEquals = (test, test)
\ No newline at end of file
+        #Act
+        test_csv = False
+        train_csv = False
+        test_rds = False
+        train_rds = False
+        for File in os.listdir(filepath):
```

Comment:
Testing these is tough, but can we get tests that at least check the lengths of the content and compare them to expected values based on the test/train ratio we are using?

Also, as much as possible please eliminate duplication in your code, see your test below.